### PR TITLE
Add seed parameter for image augmentation method.

### PR DESCRIPTION
**Pull request by hgaiser**
This PR adds a seed parameters to keras.preprocessing.image.random_transform.

The case I had required both image and corresponding segmentation to be augmented. I'm not using the flow methods or `NumpyIterator`/`DirectoryIterator`. Instead I wrote my own generator which uses `random_transform` to transform images.

Using this PR I can use a seed like the current timestamp to feed to `random_transform` to get identical transforms for my image/segmentation pair. This is analogous to the example at the bottom of [this page](https://keras.io/preprocessing/image/). However, in that case one seed suffices. In my case I have to use a different seed every time I plan to use `random_transform` on an image/segmentation pair (otherwise if I used a constant seed I'd get the same transformation applied to every pair). Not important for this PR necessarily, but it gives some additional information to people reading this.

### CODE REVIEW

**Comment by hgaiser** [comment]
Reopened PR to trigger rebuild, it appeared to have timed out before.

File: keras/preprocessing/image.py [hunk 1]
```diff
@@ -540,6 +541,9 @@ def random_transform(self, x):
         img_col_axis = self.col_axis - 1
         img_channel_axis = self.channel_axis - 1
 
+        if seed is not None:
+            np.random.seed(seed)
+
```
Comment (ozabluda) [inline keras/preprocessing/image.py]: This conditional is not necessary. You can call np.random.seed(None) unconditionally.
https://docs.scipy.org/doc/numpy/reference/generated/numpy.random.seed.html

Comment (ozabluda) [reply keras/preprocessing/image.py]: Correction - it is necessary. Without it np.random.seed(None) will re-seed. I can see the access to /dev/urandom (Numpy docs are slightly unclear).
